fjord/base/tests/test_auth.py

```diff
@@ -20,17 +22,18 @@ def test_new_user(self):
         # FIXME - this can go away post django-browserid 0.9
         new_user.backend = 'django_browserid.auth.BrowserIDBackend'
 
-        get_request = RequestFactory().get(reverse('dashboard'))
-        get_request.user = new_user
-        get_request.session = self.client.session
+        post_request = RequestFactory().post(reverse('dashboard'))
+        post_request.user = new_user
+        post_request.session = self.client.session
```

Comment:
Getting the dashboard didn't test the auth... it returned a get request which we then filled in and passed to `FjordVerify()` which did the authentication.

I'm not sure why we used the dashboard for that. That seems puzzling. When you click on "Log In", I'm pretty sure it authenticates against browserid urls.

So, I think this test was weird before and I think the changes continue the weirdness. Seems like we should fix this test.

This test covers what happens when people who don't have an account on Fjord log in with Persona. Since they're new to Fjord, they need to get passed to the `new-user-view`.